tests/desktop_history.pm

```diff
@@ -11,30 +11,41 @@ sub run {
     $self->add_nightly_sdk_repo();
     $self->install_app($app_id, run_with_nightly_sdk => 1);
 
+    # Check the startup page is shown.
     assert_screen('history_startup');
+
+    # Search for an article and load it.
     type_string('inca');
     assert_and_click('history_search_inca');
 
     # Click the back button.
     assert_and_click('history_article_inca');
-    # Enter into a category.
-    assert_and_click('history_categories');
+
+    # Enter into a random category. (The test was originally designed to enter
+    # the ‘Old Age’ category, but since the category tiles rotate regularly we
+    # can’t guarantee that.)
```

Comment:
Is there some way we can set a fixed seed so that the tiles really are always the same? eg. is it derived from the current date? Could the SDK provide a way to do this?

Reply:
(Doesn't deal with content updates, of course.)

Reply:
I think that’s something to deal with in a follow-up issue. Filed https://phabricator.endlessm.com/T25287.